Mathlib/LinearAlgebra/QuadraticForm/Basis.lean

```diff
@@ -18,7 +18,44 @@ open LinearMap (BilinMap)
 
 namespace QuadraticMap
 
-variable {ι R M N} [LinearOrder ι]
+section
+
+variable {ι R M N}
+
+variable [CommRing R] [AddCommGroup M] [AddCommGroup N] [Module R M] [Module R N] [DecidableEq ι]
+
+open Finsupp in
+theorem map_finsupp_sum (Q : QuadraticMap R M N) (f : ι →₀ R) (g : ι → R → M) :
```

Comment:
```suggestion
theorem map_finsuppSum (Q : QuadraticMap R M N) (f : ι →₀ R) (g : ι → R → M) :
```
to match the naming convention

Reply:
The linear version of this result is `map_finsupp_sum` https://github.com/leanprover-community/mathlib4/blob/6ac68693d219f89dcb242e9565245f1fbe108953/Mathlib/Algebra/BigOperators/Finsupp.lean#L198 @eric-wieser points out that the names should match the linear equivalents.